brink.sh

```diff
@@ -87,7 +86,7 @@ PYTHON_CONFIGURATION='NOT-YET-DEFINED'
 PYTHON_VERSION='not.defined.yet'
 PYTHON_PLATFORM='unknown-os-and-arch'
 PYTHON_NAME='python3.8'
-BINARY_DIST_URI='https://binary.chevah.com/production'
+BINARY_DIST_URI='unknown-dist-uri'
```

Comment:
Fine with a default location. I guess the link GitHub releases should be the default here, while bin.chevah.com should be default in `brink.conf` for now.

Also, what new script should I rename to `pythia.sh`? I don't get it, sorry… There is a script called `pythia` already.

Reply:
> Also, what new script should I rename to pythia.sh? 

My suggestion was to rename brink.sh to pythia.sh ...but maybe it's not a good idea :)

I think that pythia should be renamed `pythia_build` as it is more about build pythia.

But that is a very minor comment.

Reply:
Well, `brink.sh` would still be needed by the other repos, right? And when they are ported to Python 3, this is what they'd better use, because the changes in this repo are minimal and focused on reusability. Same for `functions.sh`, which I have simplified and resync'ed with the one in the `brink` repo while working on this repo.

As for the `pythia` script filename, I have tried different options when reassembling the `python-package` repo into the `pythia` repo. This is the filename that I found most comfortable when working in a CLI with auto-completion. I'm open to suggestions though, it's not a biggie.